content/2017-09-19_pycon-cz-conference-wrap-up-summary.md

```diff
@@ -0,0 +1,199 @@
+
+### What Could Have Been Done
+
+- Given the context we're a developer-focused conference in the Czech Republic, ⅕ of audience a good number, but in the following years we would like to find and remove as much remaining obstacles for women to attend as possible and to achieve even higher numbers. Experience from abroad shows there's still a margin.
+- [Because of last minute changes of the venue](http://blog.python.cz/pycon-cz-2017-changes-date-and-venue), both of the main conference days happened during working days. According to reports, such choice isn't exactly the best for attracting beginners.
+- For the same reason the Django Girls workshop had to be on Sunday, after the whole conference. That's why many attendees of the workshop did not come to the conference, although they were given a free ticket.
+- We did not manage to pursue more partnerships. E.g. last year we invited [VODICÍ PES](www.vycvikvodicichpsu.cz).
+
+## Code of Conduct Report
+
+The organization team agrees that all PyCon events should be accessible and welcoming to everyone, regardless of their background. We agree that a clear and specific Code of Conduct is a necessity for any event.
+
+- The [Code of Conduct](https://cz.pycon.org/2017/about/code/) was based on Conference anti-harassment policy from [Geek feminism wiki](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy) and edited to be as specific to our event as possible
+- Agreeing to the Code was mandatory when buying a ticket
+- Two contact persons (male and female) were assigned
+- Attendees were reminded of the Code in the opening speech of each day
+
+### What Could Have Been Done
+
+- Summary of the Code in both languages were not posted around the venue
+- Phone numbers of the contact persons were not posted around the venue
+- There was no easy way to report an incident other than to write an e-mail or to talk to the CoC contact person
+- Czech language version of the Code was not accessible on the website
+
+## Budget
+
+In the spirit of Open Source and transparency, we’re happy to share a summary of our budget (all numbers are thousands of Czech Koruna, CZK).
+
+### Income
+
+- Sponsorship: 648
+- Ticket sales: 400
+- **Total: 1048**
+
+### Expenses
+
+- Venue: 215
+- Catering: 187
+- Robot workshop: 9
+- Conference party (food+drinks): 41
+- Financial aid (incl. keynote speakers): 130
+- Photo/Video recording & editing: 97
+- Swag, lanyards: 29
+- On-site fun (DJ, games): 21
+- Stationery, roll-ups: 18
+- Various other: 10
+- **Total: 757**
+
+### Conclusion
+
+As you can see, we ended with a positive balance of close to 291 000 CZK. That's because the conference moved from Brno to Prague and the organization team also changed, so we were very deffensive about our estimates and spending. All the surplus funds will be applied to next year’s event budget.
```

Comment:
defensive
nejsem si jistá s tím přebytkem. pokud vím, nemáme stažený peníze z paypalu, možná tam ještě něco visí, co není započítaný? přijde mi to nějak moc přebytek :D

Reply:
Tyto čísla jsem si domyslel z Battle Planu. Ale teď už mám aktuální čísla od @martinbilek, takže je zapracuju. Další skvělý důvod proč počkat s vydáním článku na říjen! :trollface: